packages/ckeditor5-core/src/editor/editorui.js

```diff
@@ -12,8 +12,7 @@
 import ComponentFactory from '@ckeditor/ckeditor5-ui/src/componentfactory';
 import FocusTracker from '@ckeditor/ckeditor5-utils/src/focustracker';
 
-import EmitterMixin from '@ckeditor/ckeditor5-utils/src/emittermixin';
-import mix from '@ckeditor/ckeditor5-utils/src/mix';
+import { ObservableMixin, mix } from '@ckeditor/ckeditor5-utils';
```

Comment:
Revert please. It's not consistent with how the rest of the imports look here.

I started wondering though, what's the expected way to write these imports. I need to fwd this to the devops team because this deserves to be linted.

Reply:
Reported: #10375

Reply:
@oleq  made this change, not sure thee reason behind. I'll make it the old way :)